packages/core/scripts/severless-cloud-run/CloudRunHub.js

```diff
@@ -39,24 +37,29 @@ const datastore = new Datastore();
 const Web3 = require("web3");
 
 const { Logger, waitForLogger } = require("@umaprotocol/financial-templates-lib");
+let logger;
+let protocolRunnerUrl;
+let customNodeUrl;
+let hubConfig = { mode: "localStorage" };
 
-app.post("/", async (req, res) => {
+hub.post("/", async (req, res) => {
```

Comment:
is this how the IPC works between spoke and hub locally and in cloud run, all in express?

Reply:
yup! cloud run exposes a restful API interface that you call to spool up the docker instance. The hub is initialized by calling a restful API on the hub IP and port and it then calls n number of spoke API calls on the spoke interface.

Reply:
Defining it this way also means we can easily expand the logic into use cloud functions which also expose restful APIs.